sdk/keyvault/azure-security-keyvault-secrets/inc/azure/keyvault/secrets/secret_client.hpp

```diff
@@ -1,295 +1,388 @@
+     * secrets/set permission.
+     * @param secretName The name of the secret. The value you provide may be copied globally for
+     * the purpose of running the service. The value provided should not include personally
+     * identifiable or sensitive information.
+     * @param parameters The parameters for setting the secret.
      * @param context The context for the operation can be used for request cancellation.
+     * @return A secret consisting of a value, id and its attributes.
      *
-     * @return The Secret wrapped in the Response.
      */
-    Azure::Response<DeletedSecret> GetDeletedSecret(
-        std::string const& name,
-        Azure::Core::Context const& context = Azure::Core::Context()) const;
+    Response<KeyVaultSecret> SetSecret(
+        std::string const& secretName,
+        SecretSetParameters const& parameters,
+        Core::Context const& context = {}) const;
 
+    // codegen: insert after SecretClient::SetSecret
+  public:
     /**
-     * @brief Set a secret in a specified key vault.
-     *
-     * @param name The name of the secret.
+     * @brief The SET operation adds a secret to the Azure Key Vault. If the named secret already
+     * exists, Azure Key Vault creates a new version of that secret. This operation requires the
+     * secrets/set permission.
+     * @param secretName The name of the secret. The value you provide may be copied globally for
+     * the purpose of running the service. The value provided should not include personally
+     * identifiable or sensitive information.
      * @param value The value of the secret.
-     *
      * @param context The context for the operation can be used for request cancellation.
-     * @return The Secret wrapped in the Response.
+     * @return A secret consisting of a value, id and its attributes.
+     *
      */
     Azure::Response<KeyVaultSecret> SetSecret(
-        std::string const& name,
+        std::string const& secretName,
         std::string const& value,
-        Azure::Core::Context const& context = Azure::Core::Context()) const;
+        Azure::Core::Context const& context = Azure::Core::Context()) const
+    {
+      SecretSetParameters parameters;
+      parameters.Value = value;
+      return SetSecret(secretName, parameters, context);
+    };
 
     /**
-     * @brief Set a secret in a specified key vault.
-     *
-     * @param name The name of the secret.
-     * @param secret The secret definition.
-     *
+     * @brief The SET operation adds a secret to the Azure Key Vault. If the named secret already
+     * exists, Azure Key Vault creates a new version of that secret. This operation requires the
+     * secrets/set permission.
+     * @param secretName The name of the secret. The value you provide may be copied globally for
+     * the purpose of running the service. The value provided should not include personally
+     * identifiable or sensitive information.
+     * @param secret The secret to set.
```

Comment:
Are doxygen docs good?
The ones I've seen do look good, and not only good but in most cases better, but can we do something to review all of them in a diff, any ideas?
If we want to correct some of them, we can also add `// codegen: replace SecretClient::SetSecret comment` injection context.

Reply:
i don't understand the question

Reply:
Can we have a meeting like the ApiView one, but for the generated documentation?